### PR TITLE
Requesting a pull to oyiptong:sp2 from oyiptong:fix-build-task

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ncloudioj** [comment]
r+ let's ship it! @k88hudson

File: splice/web/views.py [hunk 1]
```diff
@@ -15,9 +18,9 @@ def distribution():
     return render_template('distribution.jinja2')
 
 
-@root.route('/campaign/', methods=['GET'])
+@root.route('/campaign', methods=['GET'])
 def campaign():
-    return render_template('campaign.jinja2')
+    return send_from_directory(campaign_static_path, 'index.html')
```
Comment (rlr) [inline splice/web/views.py]: much better :+1: . sorry about that :hankey: (why isn't it :poop:?)

Comment (rlr) [inline splice/web/views.py]: oh :poop: works too. whew

Comment (k88hudson) [inline splice/web/views.py]: hahaha :hankey:

**Comment by ncloudioj** [comment]
i dunno what kind of :hankey: you guys are talking about, but looks like the :hankey: is working now. :100:

**Pull request merged**